docs/data/kaivos/kaivos-account.md

```diff
@@ -1,15 +1,20 @@
 # 1. Getting a database account
```

Comment:
All numbers in headings should be removed

Reply:
Removed